### PR TITLE
Disable standalone porting

### PR DESCRIPTION
**Pull request by huawenm**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [ ] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tst/PortingAssistantExtensionIntegTests/AssessmentTests.cs [hunk 1]
```diff
@@ -96,22 +96,6 @@ public async Task TestMvcMusicStoreAsync()
             Assert.IsTrue(portResult);
         }
 
-        [Test]
-        public async Task TestMvcMusicStorePortingOnlyAsync()
```
Comment (ShanshanHe) [inline tst/PortingAssistantExtensionIntegTests/AssessmentTests.cs:100]: Maybe we can keep this test and make it a negative test, and assert the `portResult` is false?

**Review by ShanshanHe — APPROVED** [review]
LGTM with minor comment

File: tst/PortingAssistantExtensionIntegTests/AssessmentTests.cs (hunk 1 above)
Comment (huawenm) [reply tst/PortingAssistantExtensionIntegTests/AssessmentTests.cs:100]: sure, will do

File: src/PortingAssistantExtensionServer/Services/PortingService.cs [hunk 2]
```diff
@@ -59,22 +59,20 @@ public ProjectFilePortingResponse PortingProjects(ProjectFilePortingRequest requ
         {
             try
             {
+                if (ProjectPathToDetails.Count == 0)
```
Comment (saikiranakula-amzn) [inline src/PortingAssistantExtensionServer/Services/PortingService.cs:62]: check for nulls as well ProjectPathToDetails == null || ProjectPathToDetails.Count == 0

**Review by saikiranakula-amzn — COMMENTED** [review]
added comment

**Pull request closed without merging**